packages/core/src/routes/+page.svelte

```diff
@@ -526,7 +528,7 @@ const notify = useNotify();
   <div>
     <Button on:click={handleOpenModal}>Open Modal</Button>
     <Modal
-      open={modalOpen}
+      isOpen={modalOpen}
       on:close={handleCloseModal}
```

Comment:
Looks like this `on:close` can be removed

Reply:
whoops, thanks!